mmdet3d/datasets/transforms/loading.py

```diff
@@ -424,7 +424,10 @@ def __init__(
         self.load_dim = load_dim
         self.use_dim = use_dim
         self.file_client_args = file_client_args.copy()
-        self.file_client = None
+        if self.file_client_args is not None:
+            self.file_client = mmengine.FileClient(**self.file_client_args)
```

Comment:
if you remove `self.file_client = mmengine.FileCline(**self.file_client_args)` to `__init__` function, you should delete it in `_load_points`

Reply:
I think it's necessary because when self.file_client is None, it could new a file_client object in call _load_points.

Reply:
Since the dafault `file_client_args` is `dict(backend='disk')`. So typically it cannot be `None`,  thus `self.file_client` can not be `None`